Mediapipe.Net/Calculators/ICalculator.cs

```diff
@@ -27,7 +27,7 @@ public interface ICalculator<T> : IDisposable
         public ImageFrame Send(ImageFrame frame);
 
         /// <summary>
-        /// This event triggers every time MediaPipe is able to detect landmarks on a given <see cref="ImageFrame"/>.
+        /// This event is triggered every time MediaPipe is able to detect landmarks on a given <see cref="ImageFrame"/>.
```

Comment:
That's great but there's still the specificity of the landmarks. Something along the lines of "is able to provide for the secondary output"?